document floors (#39)
Kruptein/planarally-docs: docs/player/gameboard.md

Also, during combat encounters, the [initiative tool](/docs/tools/initiative) will be displayed free-floating on the gameboard.

### DM-only layers
There are two DM-only layers that players can neither directly see nor directly interact with.
On the [DM](/docs/dm/layers/#dm) layer, the DM can position NPC, monsters, assets or everything else that can be positioned on a layer, for later use.
This way, a band of bandits hidden in the woods or a group of invisible monsters does not need to be imported, placed and positioned *ad-hoc* when encountering them, but can be simply transferred to the *tokens* layer with some simple clicks.

On the [FOW](/docs/dm/layers/#fow) layer (FOW translating to Fog of War), the DM will draw walls and other barriers that hinder movement and/or light to pass.

## Floors

All of the above layers combined make up a *floor*.
This is to support vertically stacked places like dungeons or buildings.
In this cases, every level of a dungeon or building could be represented by a floor.

> You can access floors at the menue at the bottom left of the game board.

Reviewer: ```suggestion
You can access floors in the menu at the bottom left of the game board.
```

The selected floor will render on your game board, *above* all layers that are set below it.

> Note, though, that you might not be able to see everything of a layer, depending on the [lighting](/docs/dm/light-shadows) and whether you sufficient access to the tokens on this floor.

Reviewer: ```suggestion
Note, though, that you might not be able to see everything of a layer, depending on the [lighting](/docs/dm/light-shadows) and whether you have sufficient access to the tokens on this floor.
```

As the floors are rendered on top of each others, you might be able to see the floors below the floor you are present on.
A common example for this would be a window, balcony, or even a hole in the boarding.
So if there is nothing hindering your sight, you can still see some things happening on the floors below (e.g. when looking out of a window).

You are not able to change the floor you are on, yourself.
Instead, you will be transferred to other floors by the DM.

> Making players able to change floors in certain places (where, e.g. stairs are on the map) is discussed bot not yet planned to be implemented in the forseeable future.

Reviewer: This is not correct, players can change floors at will currently.  The DM can prevent some floors to be accessible by players since a recent change with the eye symbol though.

Author: I actually wanted to refer to player-controlled tokens, not the players themselves. Sorry & thanks for pointing that out. :)

> Note, that even when all your tokens (or your only token) were transferred to another floor, you would still need to change to this floor on your own.

Reviewer: This is not correct, when a token is moved, the owner will instantly be moved as well.

Author: I should file a bug-report, then. Or I check my memory.
… probably both.

A common example for this would be a window, balcony, or even a hole in the boarding.
So if there is nothing hindering your sight, you can still see some things happening on the floors below (e.g. when looking out of a window).

Currently, you can see all floors that exist on a location and can freely change to any floor there is.
However, only the DM can send tokens to other floors.

> Note, that even when all your tokens (or your only token) was transferred to another floor, you would still need to change to this floor on your own.

Author: ```suggestion
Note, that even when all your tokens (or your only token) were transferred to another floor, you would still need to change to this floor on your own.
```
